lib/SimplePie/SimplePie/File.php

```diff
@@ -54,6 +54,9 @@
  */
 class SimplePie_File
 {
+
+	const DEFAULT_HTTP_ACCEPT = 'application/atom+xml, application/rss+xml, application/rdf+xml;q=0.9, application/xml;q=0.8, text/xml;q=0.7, text/html;q=0.6, text/plain;q=0.5, application/octet-stream;q=0.1';
```

Comment:
Hold on, are you absolutely sure you want to remove */*? If Mastodon doesn't work that's Mastodon's problem. That doesn't mean we should break everything.

Reply:
Also I'd like to emphasize there's nothing "fix" about "work around Mastodon's problem".

Reply:
The thing is that we are not able to read other things, so we are actually lying when saying `*/*`.
But I do believe there is a but in Mastodon, and I might try to have a look (but not in the coming days).
We can postpone to 1.22 to give it a longer try in edge.

Reply:
How I've interpreted this header for the past decade is that the idea is to tell the server to send over its stuff and we'll give it a go, because it might be XML we can handle with some random incorrect mimetype.

Imho it's far from obvious that we should give broken Mastodon preference, because broken server configurations are usually just a minor mistake of some sort in e.g. Apache or Nginx which are otherwise not broken.

Reply:
I have reverted the change of HTTP Accept for now, especially since there seems to be an easy fix for Mastodon (only some versions seem broken), by being explicit about the RSS feed address. Example: https://framapiaf.org/@davidrevoy.rss